payments-core/api/payments-core.api

```diff
@@ -5245,6 +5245,10 @@ public abstract interface class com/stripe/android/payments/core/injection/Injec
 public abstract interface annotation class com/stripe/android/payments/core/injection/InjectorKey : java/lang/annotation/Annotation {
 }
 
+public final class com/stripe/android/payments/core/injection/InjectorKt {
```

Comment:
Does this need to be public?

Reply:
Yes, it's defined in core but needs to be accessed from payments module. Added a `@RestrictTo` to it